### DISCUSSION TITLE
The description mentions a "Server-side component." Where can I find this?

### DISCUSSION THREAD

**Discussion by youporkchop5** [VickyFrenzy/gmod-enhanced-camera-2#6] [Q&A] [answered]
I ask because the height adjustment effect doesn't seem to be working, so I'm assuming I'm missing this server side component.

**Comment by VickyFrenzy** [comment, marked as answer]
This can actually be a bit confusing and is from the original description. I'll probably rephrase that...

Nothing should be missing. What the description is saying is a case if you are playing on a server that allows for client-side scripts. Which is pretty rare.

If you're having view height issues that might be caused by another addon which is overriding the view height values.

**Reply by youporkchop5** [reply]
Any ideas as to the kinds of addons that would cause this? Only one I can think of right now that might be causing would be vmanip, but that seems unlikely to me.

**Reply by VickyFrenzy** [reply]
It's mostly playermodels that came with their own custom view height built-in or other addons that change the view height.

**Reply by youporkchop5** [reply]
I don't have any addons like that installed. The playermodels I want to be using this with don't work nor have a custom view height built-in, and neither do any of the default ones change my height. What's strange is the dynamic height adjustment setting does seem to work with crouching. I disabled it for now because it was somewhat annoying, but the actual height of my character wasn't changing even with it enabled, it was only when I crouch that the height was different for each playermodel.

**Reply by VickyFrenzy** [reply]
Is "Static view height" enabled?
It has to be enabled or else you will have default height.
Also yes dynamic height only work when crouching for some reasons...
That's why there is both static and dynamic and they are not exclusive. The static one is needed.
Anyway this still needs work.


I still think you have an addon messing with the height.

Can you provide your addon list so we can be sure about this?

I've made an handy tool that will export your addon list in Markdown: <https://steamcommunity.com/sharedfiles/filedetails/?id=2443976641>/<https://github.com/VictorienXP/gmod-dump-addon-list-in-markdown>


Could you illustrate your view height in form of screenshots/video in front of the gm_construct mirror?

**Reply by youporkchop5** [reply]
![Screenshot (236)](https://user-images.githubusercontent.com/52052551/113437184-15a3a100-93de-11eb-888e-c2354e38c41c.png)
This is the best screenshot I could get to illustrate that the view height change isn't working
![Screenshot (235)](https://user-images.githubusercontent.com/52052551/113437221-2a803480-93de-11eb-95b3-a2d386e99311.png)
Here's another one, this is a fairly short playermodel, though less obvious than the last screenshot.

[addon_list.txt.txt](https://github.com/VictorienXP/gmod-enhanced-camera-2/files/6250571/addon_list.txt.txt)
Here's my addon list using your tool.
Let me know if you can't see or download any of these.

**Reply by VickyFrenzy** [reply]
I've identified the addon that is overriding the view height.

It's the "[Shrinkinator - Player Resizer](https://steamcommunity.com/sharedfiles/filedetails/?id=2377962193)".

**Reply by youporkchop5** [reply]
yup, that was it. Thanks
